proof/invariant-abstract/RISCV64/ArchDetype_AI.thy

```diff
@@ -495,9 +495,7 @@ proof -
 qed
 
 lemma in_user_frame_eq:
-  notes [simp del] = atLeastAtMost_iff atLeastatMost_subset_iff atLeastLessThan_iff
-                     Int_atLeastAtMost atLeastatMost_empty_iff split_paired_Ex
-                     order_class.Icc_eq_Icc
+  notes [simp del] = atLeastAtMost_simps order_class.Icc_eq_Icc
```

Comment:
I like this kind of thing instead of the `blah[simp del]` seen in other places. If we're doing cleanup, would it make sense to get rid of the `blah` at some point?